debian/copyright

```diff
@@ -0,0 +1,28 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: systemfixtures
+Upstream-Contact: Free Ekanayaka <free@ekanayaka.io>
+Source: https://github.com/freeekanayaka/systemfixtures
```

Comment:
This does not seem to be correct.

Reply:
I am not sure if I can update the copyright. Any views ?

Reply:
Let's keep it for now.